nps-unit-sample.json

```diff
@@ -1,367 +1,367 @@
-{
-  "name": "Acadia National Park",
-  "params": [
-    "visibility": {
-      "summariesByYear": [
-        {
-          "year": 2000,
-          "conditionStatus": "Warrants Moderate Concern",
```

Comment:
Previously, these were cross-referenced by integer code. We can continue to do it that way, but if it's equivalent for your end, it's considerably preferable for the file to be self-evident like this.

Reply:
Self-evident works for me!